test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -298,11 +298,11 @@ func (HeaderOperations) ResponseBoolHandleResponse(resp *azcore.Response) (*Head
 	if !resp.HasStatusCode(http.StatusOK) {
 		return nil, newError(resp)
 	}
-	val, err := strconv.ParseBool(resp.Header.Get("value"))
+	Value, err := strconv.ParseBool(resp.Header.Get("value"))
```

Comment:
`Value` should be camel-cased.

Reply:
Value is capitalized because we're using the name we've parsed for the fields (exported). I could camel case it in place only when we're parsing values. But the value stored in language.go.name is capitalized since it will be used as part of the structs.

Reply:
Done, they're camel-cased now and int32 and float32 vars end with 32 instead of 0